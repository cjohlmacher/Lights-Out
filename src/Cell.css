.Cell {
  height: 100px;
  width: 100px;
  background-color: #a9a9a95e;
  cursor: pointer;
}

.Cell-lit {
  background-color: #fff700c7;
}