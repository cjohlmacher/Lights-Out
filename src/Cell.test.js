import { render } from '@testing-library/react';
import Cell from './Cell';

it('renders without crashing', () => {
    render(<Cell isLit={true}/>);
});

it('matches snapshot when lit', () => {
    const {asFragment} = render(<Cell flipCellsAroundMe={() => null} isLit={true} />);
    expect(asFragment()).toMatchSnapshot();
});

it('matches snapshot when unlit', () => {
    const {asFragment} = render(<Cell flipCellsAroundMe={() => null} isLit={false} />);
    expect(asFragment()).toMatchSnapshot();
});