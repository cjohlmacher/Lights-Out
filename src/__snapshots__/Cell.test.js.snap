// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot when lit 1`] = `<DocumentFragment />`;

exports[`matches snapshot when unlit 1`] = `<DocumentFragment />`;
